plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/AppEngineStandardWizardPage.java

```diff
@@ -67,9 +80,31 @@ public void createControl(Composite parent) {
     projectIdField.setLayoutData(projectIdPosition);
     projectIdField.addModifyListener(pageValidator);
 
+    // Manage APIs
+    addManageLibrariesWidgets(container);
+
     Dialog.applyDialogFont(container);
   }
 
+  private void addManageLibrariesWidgets(Composite container) {
+    apiGroup = new Group(container, SWT.NONE);
+    apiGroup.setText("Manage Libraries");
```

Comment:
We shoudl add message bundle in this PR then. You don't need to convert the existing messages in this package here, but let's not add to the problem.